build/ci.go

```diff
@@ -1008,8 +1009,13 @@ func doXgo(cmdline []string) {
 	flag.CommandLine.Parse(cmdline)
 	env := build.Env()
 
+	subCmd := "get"
+	if strings.HasPrefix(runtime.Version(), "go1.18") {
```

Comment:
Will you deprecate xgo in go version 1.8.x? if it will, that is ok. But if go version will be 1.19, it should be changed.

Reply:
I will before go1.19